cmd/gitops/ui/run/cmd.go

```diff
@@ -93,7 +93,7 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	if isatty.IsTerminal(os.Stdout.Fd()) {
+	if isatty.IsTerminal(os.Stdout.Fd()) && os.Getenv("SKIP_OPEN_BROWSER") == "" {
```

Comment:
:question: How would you feel about making this a CLI argument instead? That way, a user can specify the flag on the CLI and it gets printed in the help.

Reply:
I wasn't completely sure about this approach initially, but I do like this idea